app/dashboard/styles.scss

```diff
@@ -0,0 +1,227 @@
+/* stylelint-disable */
```

Comment:
I know we decided not to update this file to the new style conventions, but would you mind doing color variables here too? I'd like to be able to converge on something resembling a color palette before too long, and this page has most all the colors we actually use.

Reply:
Nope, that's a good idea.